src/pages/Nav/FindProduct.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useState } from 'react';
+import { useEffect } from 'react';
+
+const SearchId = ({ searchInput }) => {
+  const [feedData, setFeedData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/productlist.json')
+      .then(response => response.json())
+      .then(result => setFeedData(result));
+  }, []);
+
+  const searchIdData = feedData.filter(e => e.name.includes(searchInput));
+
+  if (searchInput.length > 0 && searchIdData.length > 0) {
+    return (
+      <div className="filterBox">
+        {searchIdData.map((productlist, i) => {
+          return (
+            <div key={i} className="filterId">
+              <img src={productlist.image} alt="img" className="filterImg" />
+              {productlist.name}
+              {productlist.discription}
+              {productlist.price}
+            </div>
+          );
+        })}
```

Comment:
```suggestion
        {searchIdData.map(({id, image, name, discription, price}) => {
          return (
            <div key={id} className="filterId">
              <img src={image} alt="img" className="filterImg" />
              {name}
              {discription}
              {price}
            </div>
          );
        })}
```
위와 같은 방식으로 구조분해 할당해서 사용해주시고,
key에는 id 값을 넣는 방식으로 구현해주세요!